dace/libraries/blas/nodes/axpy.py

```diff
@@ -273,14 +314,14 @@ class Axpy(dace.sdfg.nodes.LibraryNode):
 
         Implementations:
         pure: dace primitive based implementation
-        fpga_stream: FPGA implementation optimized for streaming
+        fpga: FPGA implementation optimized for both streaming and array based inputs
     """
 
     # Global properties
     implementations = {
         "pure": ExpandAxpyVectorized,
-        "fpga_stream": ExpandAxpyFPGAStreaming,
-        "IntelFPGA": ExpandAxpyIntelFPGAVectorized
+        "fpga": ExpandAxpyFPGAStreaming,
+        "Intel_FPGA_DRAM": ExpandAxpyIntelFPGAVectorized
```

Comment:
Do we still need this, or does your new version cover all cases?

Reply:
Depends, @TizianoDeMatteis version uses unrolling if I understood correctly. My thing is vectorized with Dace vectors and runs on both vendors. Don't know if you want to keep both, maybe there are use cases?

Reply:
Ok, let's keep it. Could you rename it so we remember, to something like "IntelFPGAUnrolling"

Reply:
The reason I renamed Intel_FPGA_DRAM to `IntelFPGA` is because it needs to have a standard name across the library nodes. all the others use IntelFPGA

Reply:
@manuelburger Rename this to "IntelFPGAUnrolling" then we're ready to merge

Reply:
Updated the naming back to IntelFPGA and the test is now called `intel_fpga_unroll`